custom/icds_reports/reports/awc_reports.py

```diff
@@ -967,6 +969,26 @@ def get_infa_value(data, prop):
                     'format': 'string',
                     'show_percent': False,
                     'frequency': 'month'
+                },
+                {
```

Comment:
Same here. its being returned without feature flag. Where is it being prevented from showing when feature flag is off?

Reply:
@Rohit25negi  i handled it in the UI